src/api/v1beta1/properties.go

```diff
@@ -85,6 +85,10 @@ func (dk *DynaKube) HasActiveGateTLS() bool {
 	return dk.ActiveGateMode() && dk.Spec.ActiveGate.TlsSecretName != ""
 }
 
+func (dk *DynaKube) HasProxy() bool {
+	return dk.Spec.Proxy == nil || (dk.Spec.Proxy.Value == "" && dk.Spec.Proxy.ValueFrom == "")
```

Comment:
Shouldn't this be called `HasNoProxy` ? (or something similar)

Reply:
or have change the condition to`dk.Spec.Proxy != nil && (dk.Spec.Proxy.Value != "" || dk.Spec.Proxy.ValueFrom != "")`

Reply:
everywhere this is used its negated `!sts.HasProxy()` so renaming it to `HasNoProxy` would create a double negation which is not nice 😅 

pls update the condition to match the name, and update the places where its used 🙂

Reply:
Right. I prefer "if dynakube.HasProxy then.."